tests/test_properties.py

```diff
@@ -1,17 +1,23 @@
-
 def test_properties(listener):
-    from pydispatch import Dispatcher, Property
+    from pydispatch import Dispatcher, Property, AliasProperty
     class A(Dispatcher):
         test_prop = Property('default')
         name = Property('')
         something = Property()
+
+        def get_name_something(self):
```

Comment:
Can you also add a test for the `setter`?

Reply:
> Can you also add a test for the `setter`?

I must of forgot to commit this. I already have the setter added here locally I'll fix the other issues and re-commit.